README.md

```diff
@@ -1,16 +1,93 @@
-# bars_frontend
+# EMPOWER
 
-First frontend of our app.
+An app that visualizes the effects of changing inputs on the results of given logistic regression outputs.
+It can be used for any kind of data. For the example configs, we used a health use case to predict tuberculosis, diabetes, asthma and COPD on certain parameters like age, weight, height and smoking behavior.
 
-## Getting Started
+## Steps to run
+1. Clone the project
+2. Configure
+    * Connect to a running server of [patientEmpowerment](https://github.com/KBorchar/patientEmpowerment) via editing `assets/server.conf` and load `assets/feature-config.json` and `assets/models.json` from there. 
```

Comment:
Are we migrating these to patientempowerment?

Reply:
Do you mean migrating the backend repo? I'd vote yes.

Reply:
k then this needs a TODO